RunTests.md

```diff
@@ -1,33 +1,72 @@
 # How to execute Kapua tests
 
-This is document in preparation. It contains instructions how to prepare environment for testing
-and how to execute unit, integration and functional tests. Test environment is either based on embedded servers or external setup based on Docker Compose.
+There are couple of ways to run tests, either pure unit tests, component tests or integration tests.
+Integration tests can be run:
+
+- using embedded servers
+- using dockerized servers
 
 ## With dockerized Kapua
-Start dockerized Kapua
+With fabric8 plugin usage tests can now be run without explicitly running dockerized environment.
+Docker containers providing Kapua infrastructure are started with maven itself.
+But tu run these integration tests, you have to switch to qa folder and run following command
+
+    mvn test -PI9nTests
     
-      mvn -Dcucumber.options="--tags @integration" -Dgroups='!org.eclipse.kapua.test.junit.JUnitTests' -Dcommons.settings.hotswap=true -Dorg.eclipse.kapua.qa.noEmbeddedServers=true -Dcommons.db.jdbcConnectionUrlResolver=DEFAULT -Dcommons.db.schema.update=true -Dcommons.db.connection.host=localhost -Dcommons.db.connection.port=3306 -Dcommons.db.schema=kapuadb -Dcommons.db.connection.scheme=jdbc:h2:tcp -Dbroker.host=localhost verify  
+This will run integration tests only, those are tests written in gherkin and being annotated with
+``@integration``
+
+If tests fail and dockers are still running use this command:
+
+    mvn docker:stop -I9nTests
```

Comment:
You are missing "P" in "-I9nTests" - the command should be "mvn docker:stop -PI9nTests"